README.md

```diff
@@ -0,0 +1,5 @@
+This repo has the tools developed by [Apache polaris community](https://polaris.apache.org/)
```

Comment:
Can you please add ASF header here ?

Reply:
Done. Added for both the README in the repo.